update-regression-tests

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+# This script is solely used on local
+
+GITHUB=github.com
+
+# getting tests
+git clone git@github.com:pulp-platform/regression_tests.git
```

Comment:
Comment on cloning a specific version also applies here.

Additionally, why do you clone a public repository via SSH? Does that work (did you add credentials)? Cloning via HTTPS is simpler and should be sufficient.

Reply:
Now it clones directly with HTTPS. Anyway, the gitlab server have ssh I guess, otherwise it would not have worked, as you said.